teamcity/README.md

```diff
@@ -1,22 +1,22 @@
-# Agent Check: Teamcity
+# Agent Check: TeamCity
 
 ## Overview
 
-This check watches for successful build-related events and sends them to Datadog.
-
-Unlike most Agent checks, this one doesn't collect any metrics-just events.
+This integration connects to your TeamCity server to submit metrics, service checks, and events, allowing you to monitor the health of your TeamCity projects' build configurations, build runs, server resources, and more.
 
 ## Setup
 
 ### Installation
 
-The Teamcity check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your Teamcity servers.
+The TeamCity check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your TeamCity servers.
 
 ### Configuration
 
-#### Prepare Teamcity
+#### Prepare TeamCity
 
-To prepare Teamcity, see [Enabling Guest Login][2].
+1. To prepare TeamCity, see [Enabling Guest Login][2].
+2. To collect metrics, enable `Per-project permissions` and assign the `View Usage Statistics` permission to the Guest user.
+3. To enable the check to automatically detect build configuration type during event collection, assign the `View Build Configuration Settings` permission to the Guest user. 
```

Comment:
I'm currently trying this out myself, and I'm not too sure where to set this permission on the `Authentication` page. Could you add some screenshots for this if possible?
<img width="1182" alt="image" src="https://user-images.githubusercontent.com/31313038/203613203-eeb34d28-5347-429b-a494-862306ba0123.png">